### PR TITLE
OKTA-648462 Minor Angular SPA redirect guide updates

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Fixed a couple of nits and moved code snippets to maintain guide flow.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-648462](https://oktainc.atlassian.net/browse/OKTA-648462)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
-
-2. Add the following to the `routes` array:
+1. Add the `canActivate` property to the `routes` array in `app-routing.module.ts`:
 
    ```ts
    { path: 'profile', component: ProfileComponent, canActivate: [OktaAuthGuard] }
    ```
 
-This single `/profile` route is now protected.
+The single `/profile` route is protected directly, and on launch goes directly to the sign-in page (no need to click the sign-in button).
```
Comment (alisaduncan) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md:9]: I'm not sure I understand this. Would you mind clarifying for me?

**Review by alisaduncan — COMMENTED** [review]
A few questions

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md (hunk 1 above)
Comment (alisaduncan) [reply packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md:9]: Oh gotcha. I see what you mean. Would something like this work?

> The single `/profile` route is protected directly, and on navigation will redirect you to the sign-in page if you aren't already authenticated.

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md:9]: Perfect. Thanks!

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/index.md [hunk 2]
```diff
@@ -95,13 +87,13 @@ At this point, you can move to the next step: [Creating your app](#create-app).
...
+1. Verify that the **Refresh Token** is selected as a **Grant type**. In the **Refresh Token** section, [refresh token rotation](/docs/guides/refresh-tokens/main/#refresh-token-rotation) is automatically set as the default refresh token behavior.
+   > **Note:** The default number of seconds for the **Grace period for token rotation** is set to 30 seconds. You can [change the value](/docs/guides/refresh-tokens/main/#enable-refresh-token-rotation) to any number 0–60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token. Using a value of 0 indicates that there's no grace period. However, a grace period of 0 doesn't necessarily mean that the previous refresh token is immediately invalidated. That token is invalidated after the new one is generated and returned in the response.
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/index.md:96]: Slight tweaks. I liked the original between usage, but the modified 0-60 needs more word-smithing if you keep it.

```suggestion
   > **Note:** By default, the **Grace period for token rotation** is set to 30 seconds. You can [change the rotation period](/docs/guides/refresh-tokens/main/#enable-refresh-token-rotation) to between 0 and 60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token. Using a value of 0 indicates that there's no grace period. However, a grace period of 0 doesn't necessarily mean that the previous refresh token is immediately invalidated. That token is invalidated after the new one is generated and returned in the response.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md [hunk 3]
```diff
@@ -1,16 +1,9 @@
...
-
-2. Add the following to the `routes` array:
+1. Add the `canActivate` property to the `routes` array in `app-routing.module.ts`:
 
    ```ts
    { path: 'profile', component: ProfileComponent, canActivate: [OktaAuthGuard] }
    ```
 
-This single `/profile` route is now protected.
+The single `/profile` route is protected directly, and when navigating to the page it prompts you to sign in first before routing you to the `ProfileComponent`.
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md:9]:
```suggestion
The single `/profile` route is protected directly. When you navigate to the page, you must sign in before being routed to the `ProfileComponent`.
```

Comment (alisaduncan) [reply packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md:9]: Hey @isaacdurham-okta! How would you recommend wording this if the user is already authenticated? In that case, the user isn't prompted to sign in before navigating the route.

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md:9]: I think we're trying to highlight that you can access this route if authenticated and, if not, you're prompted to sign in. How about:

_The single `/profile` route is protected directly. If you're not authenticated and navigate to this page, you must sign in before being routed to the `ProfileComponent`._ 

Is that too clunky? @alisaduncan @isaacdurham-okta

Comment (alisaduncan) [reply packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md:9]: LGTM!

Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md]: LGTM. Sorry, I missed your comments yesterday!

**Pull request merged**